apm-agent-plugins/apm-kafka-plugin/apm-kafka-headers-plugin/src/test/java/co/elastic/apm/agent/kafka/KafkaClientVersionsIT.java

```diff
@@ -44,7 +44,7 @@ public static Iterable<Object[]> data() {
             // dependency version, probably due to some static initializations done by the first one loaded.
             // We use this test runner framework only so it runs from a class loader that is not the app class loader,
             // in order to verify that we don't have class visibility problems.
-            {"2.4.0"}
```

Comment:
[question] would it be relevant to keep testing the `2.4.0` version ?

Reply:
See the comment above- we use this type of testing only to isolate the client class loading to provide more realistic runtime environment.
If we were to test more versions, we would need a separate module.
We have the `KafkaLegacyBrokerIT` for a similar purpose, that is disabled for this reason and in order to test that, you would need to not only disable it, but also use the same version in the dependencies.